confmap/resolver.go

```diff
@@ -108,7 +115,8 @@ func NewResolver(set ResolverSettings) (*Resolver, error) {
 
 	converters := make([]Converter, len(set.ConverterFactories))
 	for i, factory := range set.ConverterFactories {
-		converters[i] = factory.Create(set.ConverterSettings)
+		f := factory.Create(set.ConverterSettings)
+		converters[i] = f
```

Comment:
Is this change needed?

Reply:
No, that was from a previous attempt at something, I will revert